python/tank/platform/software_launcher.py

```diff
@@ -482,6 +482,7 @@ def get_standard_plugin_environment(self):
         - ``SHOTGUN_ENTITY_TYPE``: The current context
         - ``SHOTGUN_ENTITY_ID``: The current context
         - ``SHOTGUN_PIPELINE_CONFIGURATION_ID``: The current pipeline config id
+        - ``SHOTGUN_BUNDLE_CACHE_FALLBACK_PATHS``: Fallback bundle cache
```

Comment:
please explain where this value is coming from, not just what it is. Also pls explain with a use case why this is important to pass down and why we are doing it!